frontend/src/components/answeredAssignments/AnsweredAssignmentForm.jsx

```diff
@@ -15,143 +19,257 @@ export default function AnsweredAssignmentsForm({assignmentsDue}) {
+    const [question4, setQuestion4] = useState("");
+    const [question5, setQuestion5] = useState("");
+    const [question6, setQuestion6] = useState("");
+    const [question7, setQuestion7] = useState("");
+    const [question8, setQuestion8] = useState("");
+    const [question9, setQuestion9] = useState("");
+    const [question10, setQuestion10] = useState("");
+    const [assignmentId, setAssignmentId] = useState("");
+    
+    const getAssignmentById = (id) => {
+        console.log("Calling getAssignmentById " + id )
+        AssignmentsApi.getAssignmentById(id)
+            .then(response => {
+                console.log("Title value" +response.data.assignmentTitle);
+                setAssignmentId(response.data.id);
+                setAnsweredAssignmentTitle(response.data.assignmentTitle);
+                setAnsweredAssignmentDescription(response.data.assignmentDescription)
+                setQuestion1(response.data.question1);
+                setQuestion2(response.data.question2);
+                setQuestion3(response.data.question3);
+                setQuestion4(response.data.question4);
+                setQuestion5(response.data.question5);
+                setQuestion6(response.data.question6);
+                setQuestion7(response.data.question7);
+                setQuestion8(response.data.question8);
+                setQuestion9(response.data.question9);
+                setQuestion10(response.data.question10);
+                
+                console.log(response.data);
+            })
+    }
+    useEffect(() => {
+        
+            
+            console.log("Inside useEffect , going to call getAssignmentById " + match.params.assignId);
+            getAssignmentById(match.params.assignId);
 
-    console.log(assignmentsDue)
+        
+        },[] );
 
     const createAnsweredAssignment = (answers) => {
         console.log("Inside createAnsweredAssignment" + answers)
         AnsAssApi.createAnsweredAssignment(answers)
-        .then(()=> window.location.reload());
+        .then((response)=> {
+            console.log(response);
+            window.location.reload();
+        });
     };
+    
 
-   /* const {
-        assignmentTitle, 
-        assignmentDescription, 
-        question1, 
-        question2, 
-        question3, 
-        question4,
-        question5,
-        question6,
-        question7,
-        question8,
-        question9,
-        question10
-    } = assignmentsDue; */
-
-   // const assignmentTitles = assignmentsDue.map(assignment => assignment.assignmentTitle);
 
     return (
         <div className="card mt-4">
             <div className="card-body">
-                <h6 className="card-title" >Geography quiz</h6>
-                <p className="card-subtitle">This is a knowledge quiz to test whether you paid attention to this weeks prescribed videos!</p>
+                <h6 className="card-title" >{answeredAssignmentTitle}</h6>
```

Comment:
Should the card title not be populated with the Assignment title?

Reply:
yes currently it is displaying answeredAssignmentTitle as card title . Green line shows my changes